src/test/frontend/modules/outliner/core_test.cljs

```diff
@@ -390,7 +390,7 @@
               (recur (conj result next) next)))
           result)))))
 
-(deftest random-inserts
+(deftest ^:long random-inserts
```

Comment:
These tests take up most of the time on local runs. With metadata it's easy to exclude these e.g. `node static/tests.js -e long`